{
  try {
    executeDeallocate("my_query","DEALLOCATE PREPARE my_query",TEST_SESSION);
    fail("expected exception");
  }
 catch (  PrestoException e) {
    assertEquals(e.getErrorCode(),NOT_FOUND.toErrorCode());
    assertEquals(e.getMessage(),"Prepared statement not found: my_query");
  }
}
